client/src/workspaceFolders.ts

```diff
@@ -82,21 +92,40 @@ export class WorkspaceFoldersFeature implements DynamicFeature<undefined> {
 				id: id,
 				registerOptions: undefined
 			});
+			this.sendInitialEvent(workspace.workspaceFolders);
```

Comment:
To make this consistent with other feature (e.g. DidOpenTextDocumentFeature) could we send the extra event in the register hook. Then we always have the same code pathes.

Reply:
like so?